src/components/CurrentSong.jsx

```diff
@@ -34,6 +61,43 @@ function CurrentSong() {
                 </div>
             </div>
 
+<<<<<<< HEAD
+            <div className="current-song-right-side col-4">
+                {/* <div className="song-bar">
+                    <input type="range" min="1" max="100" step="1" value={percentage} onChange={changePercent} id="range" className="custom-range" />
+                </div> */}
+                <div className="music-play-buttons">
+                    {/* skip backwards */}
+                    {/* <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="currentColor" className="bi bi-skip-backward-circle" viewBox="0 0 16 16">
+                        <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z" />
+                        <path d="M11.729 5.055a.5.5 0 0 0-.52.038L8.5 7.028V5.5a.5.5 0 0 0-.79-.407L5 7.028V5.5a.5.5 0 0 0-1 0v5a.5.5 0 0 0 1 0V8.972l2.71 1.935a.5.5 0 0 0 .79-.407V8.972l2.71 1.935A.5.5 0 0 0 12 10.5v-5a.5.5 0 0 0-.271-.445z" />
+                    </svg> */}
+                    {/* Play */}
+                    {/* <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="currentColor" className="bi bi-play-circle" viewBox="0 0 16 16">
+                        <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z" />
+                        <path d="M6.271 5.055a.5.5 0 0 1 .52.038l3.5 2.5a.5.5 0 0 1 0 .814l-3.5 2.5A.5.5 0 0 1 6 10.5v-5a.5.5 0 0 1 .271-.445z" />
+                    </svg> */}
+
+                    {/* skip forward  */}
```

Comment:
(issue): I think this implementation of the music play buttons uses an iteration of the project where the CurrentSong didn't have the asset imports or new functionality.

I think going back and resolving the merge to use the new functionality would be best, since that has functionality for the mobile view such as opening the CurrentSongOffCanvas.
![image](https://github.com/missile720/music-player/assets/45379337/19c657cd-33be-4ae5-8a58-85a55896ac19)

Reply:
(thought): Noticed that the CurrentSong component is currently removed from the Main-window. I think it would still be good to still merge the new changes to the project, just in case we still need the CurrentSong component for the implementation of playing the local files.